spec/integration/monorepo/after/gen/A/Pods.xcodeproj

```diff
@@ -561,7 +563,7 @@ Targets:
           GCC_PREFIX_HEADER: Target Support Files/A-macOS/A-macOS-App-prefix.pch
           INFOPLIST_FILE: App/A-macOS-App-Info.plist
           LD_RUNPATH_SEARCH_PATHS: "$(inherited) @executable_path/../Frameworks"
-          MACOSX_DEPLOYMENT_TARGET: '10.6'
+          MACOSX_DEPLOYMENT_TARGET: ''
```

Comment:
hold up I am verifying why this is happening.

Reply:
le sigh, yes this is a bug in determining deployment target for app specs/test specs. Fixing in CocoaPods then will re-bump this PR.

Reply:
PR https://github.com/CocoaPods/CocoaPods/pull/9153

Reply:
This applies to all the empty deployment targets in this diff.

Reply:
fixed.